enterprise/messages.py

```diff
@@ -136,8 +113,10 @@ def add_generic_error_message_with_code(request, error_code):
             '{span_start}Please reach out to your learning administrator with '
             'the following error code and they will be able to help you out.{span_end}'
             '{span_start}Error code: {error_code}{span_end}'
+            '{span_start}username: {username}{span_end}'
```

Comment:
Do we really need to display the username?

Reply:
Normally support team shares the screenshot of the enrollment page in case of error. 
There was not any indication on the enrollment page for what username the user logged in!
**for example this page.**
![image](https://user-images.githubusercontent.com/29247391/97275112-59a2e080-1857-11eb-9536-eb123ec9f3f2.png)